app/Domains/Contact/ManageCalls/Services/CreateCall.php

```diff
@@ -33,6 +32,7 @@ public function rules(): array
             'type' => 'required|string',
             'answered' => 'nullable|boolean',
             'who_initiated' => 'required|string',
+            'emotion_id' => 'nullable|integer|exists:emotions,id',
```

Comment:
@djaiss I guess emotions are not used yet?

Reply:
Emotions are used for calls.